<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigWaveDialog</class>
 <widget class="QWidget" name="ConfigWaveDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>749</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wave</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="6,1">
   <item>
    <widget class="QTabWidget" name="tabWidget_RAO_Para">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="FreqPeriWavelength_tab">
      <attribute name="title">
       <string>FreqPeriodWavelength</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTableWidget" name="table_Freq">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>417</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>宋体</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <property name="gridStyle">
          <enum>Qt::SolidLine</enum>
         </property>
         <property name="columnCount">
          <number>3</number>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>100</number>
         </attribute>
         <row>
          <property name="text">
           <string>1</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>2</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>3</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>4</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>5</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Wave Length(m) </string>
          </property>
          <property name="textAlignment">
           <set>AlignHCenter|AlignVCenter|AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Period(s)</string>
          </property>
          <property name="textAlignment">
           <set>AlignHCenter|AlignVCenter|AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Wave Circular Freq(rad/s)</string>
          </property>
          <property name="textAlignment">
           <set>AlignHCenter|AlignVCenter|AlignCenter</set>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>38504.25</string>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string>157.0000</string>
          </property>
         </item>
         <item row="0" column="2">
          <property name="text">
           <string>0.0400</string>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string>9626.0625</string>
          </property>
         </item>
         <item row="1" column="1">
          <property name="text">
           <string>78.5000</string>
          </property>
         </item>
         <item row="1" column="2">
          <property name="text">
           <string>0.0800</string>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string>4278.2500</string>
          </property>
         </item>
         <item row="2" column="1">
          <property name="text">
           <string>52.3333</string>
          </property>
         </item>
         <item row="2" column="2">
          <property name="text">
           <string>0.1200</string>
          </property>
         </item>
         <item row="3" column="0">
          <property name="text">
           <string>2406.5156</string>
          </property>
         </item>
         <item row="3" column="1">
          <property name="text">
           <string>39.2500</string>
          </property>
         </item>
         <item row="3" column="2">
          <property name="text">
           <string>0.1600</string>
          </property>
         </item>
         <item row="4" column="0">
          <property name="text">
           <string>1540.1700</string>
          </property>
         </item>
         <item row="4" column="1">
          <property name="text">
           <string>31.4000</string>
          </property>
         </item>
         <item row="4" column="2">
          <property name="text">
           <string>0.2000</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Amplitude_tab">
      <attribute name="title">
       <string>Amplitude</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QTableWidget" name="table_Amplitude">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>宋体</family>
          </font>
         </property>
         <property name="columnCount">
          <number>1</number>
         </property>
         <column>
          <property name="text">
           <string>Amplitude(m)</string>
          </property>
          <property name="textAlignment">
           <set>AlignHCenter|AlignVCenter|AlignCenter</set>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="WaveDirectionAngle_tab">
      <attribute name="title">
       <string>Wave Direction Angle</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableWidget" name="table_WaveDirectionAngle">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>宋体</family>
          </font>
         </property>
         <property name="columnCount">
          <number>1</number>
         </property>
         <column>
          <property name="text">
           <string>Wave Direction Angle((°)</string>
          </property>
          <property name="textAlignment">
           <set>AlignHCenter|AlignVCenter|AlignCenter</set>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="3" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Group Value</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_13">
          <item row="2" column="2">
           <widget class="SelfDefLineEdit" name="lineEdit_Step"/>
          </item>
          <item row="1" column="2">
           <widget class="SelfDefLineEdit" name="lineEdit_MaxValue"/>
          </item>
          <item row="0" column="2">
           <widget class="SelfDefLineEdit" name="lineEdit_Min_Value"/>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="DeleteAllBtn_RAO">
            <property name="text">
             <string>Clear All</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QPushButton" name="AddGroup_RaoNew">
            <property name="text">
             <string>Add Group</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_MaxValue">
            <property name="text">
             <string>MaxValue</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_MinValue">
            <property name="text">
             <string>MinValue</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_Step">
            <property name="text">
             <string>Step</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLabel" name="label_Step_Unit">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="label_MaxValue_Unit">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="label_MinValue_Unit">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lblwaveangle">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>:/images/waveangle.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="topMargin">
           <number>11</number>
          </property>
          <property name="verticalSpacing">
           <number>7</number>
          </property>
          <item row="0" column="0">
           <widget class="QRadioButton" name="AmplitudeRadio">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>Amplitude(m)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_7">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QRadioButton" name="directionRadio">
            <property name="text">
             <string>Wave Direction Angle(°)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>7</number>
        </property>
        <property name="topMargin">
         <number>11</number>
        </property>
        <item>
         <widget class="QRadioButton" name="wavelengthRadio">
          <property name="text">
           <string>Wave Length(m)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="periodRadio">
          <property name="text">
           <string>Period(s)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="freqRadio">
          <property name="text">
           <string>Wave Circular Freq(rad/s)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QGroupBox" name="groupBox_SingleValue">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Single Value</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_139">
          <item>
           <widget class="QLabel" name="label_InputValue">
            <property name="text">
             <string>Input Value</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="SelfDefLineEdit" name="lineEdit_InputValue"/>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_147">
          <item>
           <widget class="QPushButton" name="AddBtn_RAO">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ModifyBtn_RAO">
            <property name="text">
             <string>Modify</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="DeleteBtn_RAO">
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>tabWidget_RAO_Para</zorder>
  <zorder></zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SelfDefLineEdit</class>
   <extends>QWidget</extends>
   <header>selfdeflineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
